src/components/AddPaymentMethodMenu.js

```diff
@@ -78,13 +66,11 @@ function AddPaymentMethodMenu({isVisible, onClose, anchorPosition, anchorAlignme
                           {
                               text: translate('common.personalBankAccount'),
                               icon: Expensicons.Bank,
-                              onSelected: () => {
-                                  onItemSelected(CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT);
-                              },
+                              onSelected: () => onItemSelected(CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT),
                           },
                       ]
                     : []),
-                ...(!ReportActionsUtils.hasRequestFromCurrentAccount(lodashGet(iouReport, 'reportID', 0), lodashGet(session, 'accountID', 0))
+                ...(ReportUtils.isExpenseReport(iouReport)
```

Comment:
This is not desired fix, we need this method in place so we can show the Bussiness bank account option in case of IOUs when there is only "one direction"

Reply:
Ah I see! Thanks for clarifying!